### PR TITLE
Docs-update-link

### PR DESCRIPTION
**Pull request by bmckercher123**
## Update footer for Training

- Request from Marketing: https://tigera.atlassian.net/browse/DOCS-246.
- Merge to /nightly, cherrypick to 3.19.1
- Erica approved

### CODE REVIEW

**Review by bcreane — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
